### DISCUSSION TITLE
Add polyfill Array.at

### DISCUSSION THREAD

**Discussion by pstachula-dev** [vercel/next.js#44148] [Ideas]
### Describe the feature you'd like to request

`Array.at`  - polyfill in `core-js`
This is a frequently used Javascript feature, currently not supported on older versions of iOS

### Describe the solution you'd like

Please add polyfill for `Array.at` 
[es.array.at.js](https://github.com/zloirock/core-js/blob/master/packages/core-js/modules/es.array.at.js)

In the `next-polyfill-nomodule`
https://github.com/vercel/next.js/blob/canary/packages/next-polyfill-nomodule/src/index.js

### Describe alternatives you've considered

I don't have alternatives.

**Comment by bertho-zero** [comment]
Why has this issue been converted to a discussion? (from https://github.com/vercel/next.js/issues/44141)

**Comment by bertho-zero** [comment]
Will be fixed with https://github.com/vercel/next.js/pull/44436

**Comment by timrae** [comment]
I'm curious why does next.js need to maintain a list of polyfills in the first place? Isn't this something which would be handled automatically by babel? Sorry if this is a n00b question... I'm very new to web development and would like to understand a bit more deeply the gotchas around using modern JS features in my code and still be supported on older browsers. For example are there are any other missing polyfills besides `Array.at`?

**Reply by GabenGar** [reply]
Probably so they could version control them instead of debugging other people's babel config problems forever.
